core/chains/evm/types/nonce.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+import "fmt"
+
+var _ fmt.Stringer = &Nonce{}
+
+// Nonce wraps an EVM nonce into a stringable type
+type Nonce struct {
+	N int64
+}
+
+func (n Nonce) String() string {
+	return fmt.Sprint(n.N)
```

Comment:
nit/ unless this is on a hot path:
```suggestion
	return strconv.FormatInt(n.N, 10)
```

Reply:
Done.